lessons/Algebra2/linear4-other-forms/langs/en-us/index.adoc

```diff
@@ -37,36 +36,57 @@
 
 |===
 
-== Other Forms of Linear Models
-@objective{linear-forms}
+== Optional: Horizontal Shift in Linear Functions
```

Comment:
Why optional?

Reply:
move this to quadratic lesson where horizontal shift is introduced

Reply:
@schanzer I have started by literally cutting and pasting the section from linear to quadratic so that we can see it.

1.  I notice that horizontal shift is also optional in the quadratic lesson... so I want to check in before I do any further work.
2. Making the change real will require updating Desmos files for both lessons.